src/components/Countries/Countries.jsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import styles from './Countries.module.css';
+
+const Countries = ({ countriesData, selectedCountry, setSelectedCountry }) => {
+	return (
+		<>
+			<div>
+				<select
+					className={styles.country_list}
+					onChange={(e) => setSelectedCountry(e.target.value)}
+					required
+				>
+					<option defaultValue={''} hidden>
```

Comment:
Aktualnie mogę dodać wycieczkę bez kraju docelowego. Z tego, co wyczytałam tutaj: [https://react.dev/reference/react-dom/components/select](url) to jest kontrolowany komponent, więc zamiast `defaultValue={''}` powinno być `value=''` i z tą zmianą działa mi walidacja pustego selecta.

Reply:
poprawione